MetaMask/metamask-extension
# 17873 ledger instruction banner

## Explanation

Currently the LedgerInstructionField is using the old banner style. Seeing it next to the new BannerAlert creates inconsistencies. It would be great to update it to use BannerAlert

## Screenshots/Screencaps

### Before

![image](https://user-images.githubusercontent.com/82925669/222015191-9b3bdf8e-a2a9-442c-96ca-41e22332771d.png)

### After

![image](https://user-images.githubusercontent.com/82925669/224206438-614fd3e2-e266-4180-a6d7-246af7555a5d.png)

![image](https://user-images.githubusercontent.com/82925669/224206208-063586c4-b16e-47b5-a5a3-c3fc551b9703.png)



## Manual Testing Steps

Sign data using the E2E Test Dapp:

1. Open the E2E Test Dapp (https://metamask.github.io/test-dapp/) using a Ledger account
2. Click the "Personal Sign" option
3. The Ledger Instructions Field component should appear above the action buttons

Styling can also be viewed via the storybook using the new Components / App / LedgerInstructionField component.

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
